pkg/kubectl/cmd/config/set.go

```diff
@@ -159,6 +160,23 @@ func modifyConfig(curr reflect.Value, steps *navigationSteps, propertyValue stri
 		actualCurrValue.SetString(propertyValue)
 		return nil
 
+	case reflect.Slice:
+		if steps.moreStepsRemaining() {
+			return fmt.Errorf("can't have more steps after bytes. %v", steps)
+		}
+		innerKind := actualCurrValue.Type().Elem().Kind()
+		if innerKind != reflect.Uint8 {
+			return fmt.Errorf("Unrecognized slice type. %v", innerKind)
+		}
+
+		val, err := base64.StdEncoding.DecodeString(propertyValue)
+		if err != nil {
+			val = []byte(propertyValue)
```

Comment:
I think you should return the error here rather than silently using the raw value.